arenadata/readme.md

```diff
@@ -3,7 +3,11 @@
 ## Build docker gpdb image with developer options
 
 Change directory to gpdb sources destination. Make sure that directry doesn't contain binary objects from previous builds. Then run:
-
+for Ubuntu:
+```bash
+docker build -t gpdb7_u22:latest -f arenadata/Dockerfile.ubuntu .
```

Comment:
I ran this command and got this error:
```sh
...
make[4]: *** [../../../../../src/Makefile.mock:25: cdbsenddummypacket-check] Aborted (core dumped)
make[3]: *** [../../../../src/backend/common.mk:50: unittest-check-local] Error 2
make[2]: *** [../../../src/backend/common.mk:47: unittest-check-motion-recurse] Error 2
make[1]: *** [common.mk:47: unittest-check-cdb-recurse] Error 2
make: *** [GNUmakefile:233: unittest-check] Error 2
The command '/bin/sh -c bash /home/gpadmin/gpdb_src/concourse/scripts/compile_gpdb.bash' returned a non-zero code: 2
```

Reply:
Have you run it on a clean source tree without prebuilt binaries? What flavor of linux have run it on?

Reply:
> Have you run it on a clean source tree without prebuilt binaries?

Yes, before launching I ran the command `make -j"$(nproc)" clean`




> What flavor of linux have run it on?

hub.adsw.io/library/docker:20.10.17-x64 - as on our CI

Reply:
On a host with Ubuntu 24.04 the same command was executed successfully. I think the problem above is related to the fact that there is no ipv6 in docker.

Reply:
It is your local problem, i have this problem too. Also I have this problem at building container with Rocky Linux. On CI there is not this problem. You can disable unit test for building image.

Reply:
I have created a separate task on this issue.